src/validators/createEqualTo.js

```diff
@@ -0,0 +1,18 @@
+import i18n from '@dhis2/d2-i18n'
+import { isEmpty, requireArgument } from './helpers/index.js'
+
+const createEqualTo = (key, description) => {
+    requireArgument(key, 'string')
+
+    const errorMessage = i18n.t(
+        'Please make sure the value of this input matches the value in "{{otherField}}".',
+        { otherField: description || key }
+    )
+
+    return (value, allValues) =>
+        isEmpty(value) || isEmpty(allValues[key]) || value === allValues[key]
```

Comment:
We could change this line to just `value === allValues[key]`. If both values are empty, this will return undefined. If the original value is not empty, this should throw an error. I think this validator implies that the value of its field is required when the other value is not empty, otherwise this wouldn't make much sense.

Reply:
I think it's good you flagged this one up, but in https://github.com/dhis2/ui-forms/pull/164/commits/1d1094d840796f268c60a726ce963bb1dfaeff40 I have implemented it slightly differently than you suggested. I've kept the `isEmpty` check on the field itself, because:
- Theoretically both fields can have a different value when "being empty". Once could be an empty string, and the other be `null`
- Regardless of how likely this is too happen, and if we should allow this to happen, I like to keep all validators consistent, in the sense that they return `undefined` when the field is empty. And I don't want this validator to possibly have some different edge cases in that regard.

If you agree, please resolve this conversation.